demos/projects/ESPRESSIF/az-nvs-cert-bundle/README.md

```diff
@@ -13,5 +13,20 @@ idf.py --no-ccache -B "C:\esptrustbundle" build
 To flash, run the following:
```

Comment:
It won't let me add it earlier in the file, but our other readme's usually have a line like
``navigate to the `demos/projects/ESPRESSIF/az-nvs-cert-bundle` directory.``
which would be helpful!